apps/web/src/types/modules.d.ts

```
@@ -0,0 +1,53 @@
+declare module 'react' {
+  export function useState<T>(initial: T): [T, (v: T) => void];
+  export function useEffect(fn: () => void | (() => void), deps?: any[]): void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)